pytorch_vision_deeplabv3_resnet101.md

```diff
@@ -35,7 +38,7 @@ So, `output['out']` is of shape `(N, 21, H, W)`. More documentation can be found
 ```python
 # Download an example image from the pytorch website
 import urllib
-url, filename = ("https://github.com/pytorch/hub/raw/master/images/dog.jpg", "dog.jpg")
+url, filename = ("https://github.com/pytorch/hub/raw/master/images/deeplab1.png", "deeplab1.png")
```

Comment:
We are actually showing image and output of this file but previously were demonstrating on dog.
I think we should demo on the same image.
This is also in Colab notebook, which would need update to pytorch.github.io repo.

I have rolled this change for all 3 segmentation model.s